packages/configs/README.md

```diff
@@ -92,37 +92,37 @@ To use the `eslint` configs in an Amplify UI package, declare `eslint-config-amp
 
 > Note the dependency version of `0.0.0`. Rather than publishing our custom `eslint` configs to NPM, we utilize the `workspaces` field of the top level [_package.json_](../../package.json) to allow our packages to use the configs during development.
 
-**`eslint-config-amplify-ui`/`eslint-config-amplify-ui/react`**
+**`@aws-amplify/eslint-config-amplify-ui`/`@aws-amplify/eslint-config-amplify-ui/react`**
 
-Add the desired config to the `extends` field of the desired package _.eslintrc.js_, along with the noted `parserOptions`. The configs can be extended using either short (`'amplify-ui'`) or long (`'eslint-config-amplify-ui'`) form syntax:
+Add the desired config to the `extends` field of the desired package _.eslintrc.js_, along with the noted `parserOptions`. The configs can be extended using either short (`'@aws-amplify/amplify-ui'`) or long (`'@aws-amplify/eslint-config-amplify-ui'`) form syntax:
```

Comment:
Is this true? 

```
... The configs can be extended using either short (`@aws-amplify/amplify-ui'`)
```

I don't think the short form supports the scope does it?

Reply:
That's true! It does! 🙈

Reply:
🤯